ansible/roles/cuda/tasks/main.yaml

```
@@ -68,15 +52,15 @@
 - name: Add PATH to .bashrc
   ansible.builtin.lineinfile:
     dest: ~/.bashrc
-    line: export PATH="/usr/local/cuda/bin:$PATH"
+    line: export PATH="/usr/local/cuda-12.3/bin:$PATH"
```

```suggestion
    line: export PATH="/usr/local/cuda/bin:$PATH"
```

We can keep them same since the path points to the latest cuda installation by default.

For example on my machine:
```console
$ readlink -f /usr/local/cuda
/usr/local/cuda-12.3
```